installation/docker.md

```diff
@@ -54,15 +54,15 @@ Jason, one of our volunteers, found [a trick](https://github.com/docker/for-mac/
 
 For performance boost, there's a a popular tool called [docker-sync](http://docker-sync.io/).
 
-## Fork repositories
+## Clone repositories
 
-We will now download `ubyssey.ca` and `dispatch` projects. Follow [our forking instructions](/installation/forking-the-repo.md) first to copy the project under your GitHub username. Run these commands inside `ubyssey-dev` dir (note: to hcheck which dir you are in, try `pwd` command):
+We will now download `ubyssey.ca` and `dispatch` projects. This is the github repository (https://github.com/ubyssey). 
```

Comment:
That's the URL for a user, not for a repo. Repos have the filename .git

Reply:
Updated